### ISSUE TITLE
How to add more rows per page and fix rendering - ReactJs

### ISSUE THREAD

**Issue by gmumdzhiev** [mbrn/material-table#1009]
I am trying to apply more rows per page to my table from material-table . Until now i found out that you can add an array with with the wishful rows per page ex. rowsPerPageOptions={[5, 10, 25, 50, 100]} , but my problem is when i apply the row of 100 the table extends to an blank one . ( i receive at the moment only 24 rows (documents) from the backend ) So basicly it has to be limitted to the data that i have . Can some one give me a hand ? Thank you

```
render() {
  components={{
              Pagination: props => (
                  <TablePagination
                      {...props}
                      rowsPerPageOptions={[5, 10, 25, 50,100]}
                      count={this.state.items.length}
                      />
    ),
```

**Comment by Domino987** [comment]
Here is the [fix](https://stackoverflow.com/questions/57568738/material-table-how-to-add-more-rows-per-page-and-fix-rendering-reactjs/57569050?noredirect=1#comment101598852_57569050).

**Comment by gmumdzhiev** [comment]
Thank you !
